controller/helper/process-survey-instances.js

```diff
@@ -6,33 +6,62 @@
 
 const moment = require('moment');
 
+const sqlDateFormat = 'ddd MMM DD YYYY HH:mm:ss ZZ';
+const viewDateFormat = 'MM-DD-YYYY HH:mm';
+
 /**
  * Takes in a Survey Instances and processes it to get Complience chart details
  * @param {Array<Object>} surveys - list of survey instances
  * @returns {Object} Complience chart data
  */
 function processSurveyInstances (surveys) {
+    const filterSurveyByState = surveys.filter((survey) => {
+        return survey.state === 'completed';
+    });
+
     return {
         // Using UTC as the date gets modified to the local time (GMT in this case) when UTC not used.
-        labels: surveys.map((survey) => {
-            return moment(survey.startTime).utc().format('MM/DD/YYYY HH:mm');
-        }),
-        datasets: pickTimeLeft(surveys)
+        labels: pickDates(filterSurveyByState),
+        datasets: pickTimeLeft(filterSurveyByState)
     };
 }
 
 /**
  * Takes in a Survey Instances and get the % time left to be shown on complience chart
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} processed list of datetimes
+ */
+function pickDates (surveys) {
+    const dates = surveys.map((survey) => {
+        return moment(survey.startTime, sqlDateFormat).utc().format(viewDateFormat);
+    });
+
+    let lastDate = '';
+    const missingValue = -1;
+
+    if (surveys[0]) {
+        lastDate = moment(surveys[0].dateCompleted, sqlDateFormat).utc().format(viewDateFormat);
+    }
+
+    if (lastDate !== '' && dates.indexOf(lastDate) <= missingValue) {
+        dates.push(lastDate);
+    }
```

Comment:
@ChristianMurphy  You are right. Based on discussion with manohara. I think redundant check (line 46-48) of dates is not needed. I will remove it.